manual/README.md

```diff
@@ -13,7 +13,7 @@ In the following section the modeling options are explained per topic.
 
 ## Integration
 - [PowerDeComposer](./PowerDeComposer.md)
-- [Configure the XML export](./ModelExportConfiguration.md)
+- [MDDE model configuration](./ModelExportConfiguration.md)
```

Comment:
I think the md file reference should be changed to:
./MDDE Model configuration.md